tests/integration_test/services_test.py

```
@@ -2298,6 +2304,265 @@
+    # Arrange
+    event_url = urljoin(
+        Utility.environment["events"]["server_url"],
+        f"/api/events/execute/{EventClass.upload_file_handler}",
+    )
+    responses.add(
+        "POST",
+        event_url,
+        json={"success": True, "message": "Event triggered successfully!"},
+    )
+    file = {
+        "file_content": ("Salesstore.csv", open("tests/testing_data/file_content_upload/Salesstore.csv", "rb"))
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)